README.md

```diff
@@ -132,6 +132,7 @@ Below is a list of displays currently implemented in the library. The Omni Devic
 | | [Inky wHAT Red/Black/White](https://shop.pimoroni.com/products/inky-what?variant=13590497624147) | inky.what_red | black, red |
 | | [Inky wHAT Yellow/Black/White](https://shop.pimoroni.com/products/inky-what?variant=21441988558931) | inky.what_yellow | black, yellow |
 | | [Inky wHAT Black/White](https://shop.pimoroni.com/products/inky-what?variant=21214020436051) | inky.what_black | black |
+| | [Inky wHAT AutoDetect](https://shop.pimoroni.com/search?q=inky) | inky.auto | black, yellow, red, colour |
```

Comment:
Please use "color" vs "colour" here. There are modes for other devices using the US spelling and it will avoid confusion in the documentation.